### DISCUSSION TITLE
米游社的某位大佬的，被删掉了

### DISCUSSION THREAD

**Discussion by L502567** [share121/inter-knot#1884] [其他]
![Screenshot_2024-06-29-18-17-18-60_40d30947b37419455897df6aa3094b56](https://github.com/user-attachments/assets/d925e4c7-6c8b-4882-a734-3612520b20b8)

**Comment by GanlinSama** [comment]
为什么删啊？好可惜

**Reply by L502567** [reply]
不知道，挺好看的

**Comment by 566001** [comment]
感觉像喵斯快跑的风格

**Comment by AnbulesPeo** [comment]
这个好看！
